Source/Extensions/UILabel+Square1.swift

```diff
@@ -30,14 +30,13 @@ public extension UILabel {
   ///   - color: `UIColor` of string.
   ///   - range: The content `String`.
   func setColor(color: UIColor, range string: String) {
-    guard let text = text else {
+    guard let text = text,
+      let newAttributedString = attributedText?.mutableCopy() as? NSMutableAttributedString else {
```

Comment:
Ok, but what happens if the label doesn't have any attributed text? 
I think the method should be able to replace the normal text with the attributed one is the later is not present.